### PR TITLE
feat(developer): add flick and multitap to touch layout schema and doc 🙊

### PR DESCRIPTION
**Pull request by mcdurdin**
Does not tighten the spec for legacy data -- that should probably be a separate schema file.

See also #5029 (C5029.1 File Formats: .keyman-touch-layout changes). This doesn't 100% match what we have in #5029, so perhaps I should go back and update that, but would prefer eyes on this before I update that feature spec.

@keymanapp-test-bot skip

### CODE REVIEW

File: common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    },
+
+    "key-id": {
+      "type": "string",
+      "pattern": "^[TKUtku]_[a-zA-Z0-9_]+$"
+    },
+
+    "key-sp": {
+      "type": "integer",
+      "enum": [0, 1, 2, 8, 9, 10],
```
Comment (jahorton) [inline common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json:101]: 3 and 4 are also supported.  Those are for use of a keyboard-specific special OSK font for the key caps, I believe.

Comment (mcdurdin) [reply common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json:101]: Curious -- they are not listed anywhere in Keyman Developer:

https://github.com/keymanapp/keyman/blob/ec690f47f9979a9e8144dd3a9f1b06a4c820096f/developer/src/tike/xml/layoutbuilder/builder.xsl#L84-L91

By the looks of this, it seems that 3 and 4 are reserved by KeymanWeb as a rewiring of 1 & 2:

https://github.com/keymanapp/keyman/blob/4f36c3c2df0b14ec2d5fac36714c4e8bda54961c/common/web/keyboard-processor/src/keyboards/activeLayout.ts#L292-L306

I also see that KMW is treating `sp` as a string whereas this spec is forcing it to an integer. Given that we need to make sure that this is backwardly compatible for compiled keyboards, do we need a "legacy.spec.json" that covers this difference off along with any other similar fields (`pad`, `width`, I am looking at you)?

I don't mind the source format diverging from the compiled format; there are various things we can do to optimise the compiled keyboards if we go this way.

Comment (jahorton) [reply common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json:101]: I've made this mistake before; there _are_ keyboards where it matters.  Please refer to #4282.

Comment (mcdurdin) [reply common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json:101]: The keyboard itself isn't using `"sp": "3"` or `"sp": "4"`. Those are values assigned internally within KeymanWeb. Thus, I don't think we need to support those values in the schema.

**Review by jahorton — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 16.0.26-alpha](https://keyman.com/downloads/releases/alpha/16.0.26)

**Pull request merged**